data/templates/styles.html

```diff
@@ -1,7 +1,9 @@
 $if(document-css)$
-html {$if(linestretch)$
+html {
+$if(linestretch)$
   line-height: $linestretch$;$endif$
```

Comment:
I think this `$endif$` can go on the next line without messing up spacing (might need testing).  If so, this looks better:
```
$if(linestretch)$
   line-height: $linestretch$;
$endif$
```
Same approach for fontsize and the others.